### DISCUSSION TITLE
v0.8.0-beta1

### DISCUSSION THREAD

**Discussion by laipz8200** [langgenius/dify#7985] [Releases]
## ✨ What’s New in v0.8.0-beta1?✨

We are excited to announce the release of Dify 0.8.0-beta1! This version introduces a significant enhancement to workflow performance by enabling the parallel execution of nodes within workflows. This improvement allows workflows to run faster and more efficiently, optimizing the use of resources and reducing overall processing time.

### 🚀 Key Features

- **Parallel Execution of Nodes in Workflows**: Nodes can now be executed in parallel within a workflow, greatly increasing the execution speed. This feature is especially beneficial for complex workflows that involve multiple steps or processes, allowing for quicker completion times and improved performance.
  
  ![image](https://github.com/user-attachments/assets/98b209a1-d72e-4e8b-8bfe-7ca0babf27c4)

### Quick Start

To upgrade to this version, follow these steps:

1. Clone the repository using the following command:

   ```bash
   git clone https://github.com/langgenius/dify.git
   ```

2. Switch to the 0.8.0-beta1 branch:

   ```bash
   git checkout 0.8.0-beta1
   ```

3. Then following the [Deploy with Docker Compose > Starting Dify](https://docs.dify.ai/getting-started/install-self-hosted/docker-compose#starting-dify).

We hope you enjoy the new features and improvements in this release. As always, we welcome your feedback and contributions to make Dify even better!

<hr /><em>This discussion was created from the release <a href='https://github.com/langgenius/dify/releases/tag/0.8.0-beta1'>v0.8.0-beta1</a>.</em>

**Comment by z1ouhan** [comment]
6666666666666666

**Comment by tobegit3hub** [comment]
Great!

**Comment by Allamss** [comment]
😘
